### ISSUE TITLE
Upgrade Inventree droplet from 0.11.0 to newer version

### ISSUE THREAD

**Issue by masteradit** [invenhost/inventree_droplet#9]
It seems like newer versions of Inventree have been released, but the Digital Ocean page shows version 0.11.0. Is there a way to upgrade it?

**Comment by matmair** [comment]
The image points to stable. The first update / upgrade will pull the newest stable

**Comment by masteradit** [comment]
> The image points to stable. The first update / upgrade will pull the newest stable

I just installed a fresh instance of InvenTree, and it showed version 0.11.0. Here are the steps I followed:

1. Go to https://marketplace.digitalocean.com/apps/inventree. Even this page shows `Version 0.11.0`:
<img width="371" alt="Screenshot 2024-05-11 at 3 03 37 AM" src="https://github.com/invenhost/inventree_droplet/assets/39951503/bede302b-245a-4517-a259-18d367d28460">

2. Click `Create InvenTree Droplet` button.
3. Fill out the form and click on `Create Droplet`.
4. After it is deployed, click `Get Started` -> `Quick access to InvenTree`.
5. Login using the username and password from the console in Digital Ocean.
6. Click on `admin` on the top right corner and then click on `About InvenTree`. The popup also shows version 0.11.0:
<img width="1139" alt="Screenshot 2024-05-11 at 3 10 48 AM" src="https://github.com/invenhost/inventree_droplet/assets/39951503/2f5bbc25-3759-46ac-8c40-d935911ce700">

**Comment by matmair** [comment]
As I said, the first update / upgrade will pull the newest version

**Comment by masteradit** [comment]
Oh, I am sorry. I read it as the first installation will pull the newest version.

Is there any way I can manually update or upgrade it or do I need to wait for something? I tried searching everywhere, but I can't seem to find a solution.

**Comment by matmair** [comment]
Just normal Debian/Ubuntu. apt update && apt upgrade

**Comment by masteradit** [comment]
Okay, thanks. I ran `sudo apt update` and `sudo apt upgrade`. After the two commands executed completely, I went back to my server and it showed `502 Bad Gateway` error:
<img width="297" alt="Screenshot 2024-05-11 at 3 34 37 AM" src="https://github.com/invenhost/inventree_droplet/assets/39951503/f63e480c-1721-4d6e-ba00-7a9b5eacc8be">

**Comment by masteradit** [comment]
I saw [this comment](https://www.reddit.com/r/InvenTree/comments/1326fwx/comment/ji4733y/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) on a Reddit post and deleted the `maintenance_mode_state.txt` file as well (though it was empty). It is still stuck on the same error.

(I'll update this comment if I find a solution)

**Comment by manan-msqrd** [comment]
Hi, even I'm facing the same issue any update on this @masteradit @matmair ??

**Comment by masteradit** [comment]
A quick update:

I found [this discussion](https://github.com/inventree/InvenTree/discussions/6109) on the inventree/InvenTree GitHub repo. I ran these commands and it deployed a fresh instance of InvenTree version 12.8:
*(Source: https://github.com/inventree/InvenTree/discussions/6109#discussioncomment-7901820)*
```
sudo apt purge inventree
sudo rm -rf /opt/inventree
sudo rm -rf /home/inventree
sudo rm -rf /etc/inventree
sudo rm /etc/nginx/sites-enabled/inventree.conf
sudo apt install inventree=0.12.8-1698964278.a7487ff8.focal
```
However, when I ran `sudo apt update` and `sudo apt upgrade` again to update InvenTree to the latest version, it again gave the same 502 error.

I found another comment on the same discussion (https://github.com/inventree/InvenTree/discussions/6109#discussioncomment-8118352), which mentioned that this might be because newer versions of InvenTree require Python 3.9 or above. So I checked which Python version I have on my server and found out that I have Python 3.8.10.
